R-package/DESCRIPTION

```diff
@@ -26,7 +26,6 @@ Suggests:
     knitr,
     rmarkdown,
     imager,
-    roxygen2
```

Comment:
This is due to the conflicts in this PR. The roxygen2 has removed by me last week. The latest version of roxygen2 broke the package building.